tests/src/JIT/HardwareIntrinsics/X86/Sse2/SetVector128.cs

```diff
@@ -0,0 +1,303 @@
+                            Console.WriteLine($"{nameof(Sse2)}.{nameof(Sse2.SetVector128)} failed on ulong: expected PlatformNotSupportedException exception.");
+                        }
+                        catch (PlatformNotSupportedException)
+                        {
+
+                        }
+                    }
+
+                    for (int i = 0; i < testsCount; i++)
+                    {
+                        Span<int> value = intTable.GetAssignmentData(i).Span;
+                        Vector128<int> result = Sse2.SetVector128(value[3], value[2], value[1], value[0]);
+                        intTable.SetOutArray(result, i);
+                    }
+
+                    for (int i = 0; i < testsCount; i++)
+                    {
+                        Span<uint> value = uintTable.GetAssignmentData(i).Span;
+                        Vector128<uint> result = Sse2.SetVector128(value[3], value[2], value[1], value[0]);
+                        uintTable.SetOutArray(result, i);
+                    }
+
+                    for (int i = 0; i < testsCount; i++)
+                    {
+                        Span<short> value = shortTable.GetAssignmentData(i).Span;
+                        Vector128<short> result = Sse2.SetVector128(value[7], value[6], value[5], value[4], value[3], value[2], value[1], value[0]);
+                        shortTable.SetOutArray(result, i);
+                    }
+
+                    for (int i = 0; i < testsCount; i++)
+                    {
+                        Span<ushort> value = ushortTable.GetAssignmentData(i).Span;
+                        Vector128<ushort> result = Sse2.SetVector128(value[7], value[6], value[5], value[4], value[3], value[2], value[1], value[0]);
+                        ushortTable.SetOutArray(result, i);
+                    }
+
+                    for (int i = 0; i < testsCount; i++)
+                    {
+                        Span<sbyte> value = sbyteTable.GetAssignmentData(i).Span;
+                        Vector128<sbyte> result = Sse2.SetVector128(value[15], value[14], value[13], value[12], value[11], value[10], value[9],
+                            value[8], value[7], value[6], value[5], value[4], value[3], value[2], value[1], value[0]);
+                        sbyteTable.SetOutArray(result, i);
+                    }
+
+                    for (int i = 0; i < testsCount; i++)
+                    {
+                        Span<byte> value = byteTable.GetAssignmentData(i).Span;
+                        Vector128<byte> result = Sse2.SetVector128(value[15], value[14], value[13], value[12], value[11], value[10], value[9],
+                            value[8], value[7], value[6], value[5], value[4], value[3], value[2], value[1], value[0]);
+                        byteTable.SetOutArray(result, i);
+                    }
+
+                    CheckMethodSpan<double> checkDouble = (Span<double> x, Span<double> y, Span<double> z, Span<double> a) =>
```

Comment:
What are `y` and `a` for?

Reply:
I have reused some of the test API instead of creating additional methods and for this particular case reused method has too many arguments.